gulpfile.js

```diff
@@ -21,7 +21,9 @@ const child_process = require('child_process');
 const generateRtlCss = require('./scripts/generate_rtl_css.js');
 const gulp = require('gulp');
 const gulpif = require('gulp-if');
-const gutil = require('gulp-util');
+const log = require('fancy-log');
```

Comment:
Should we just use `console.log`?

Reply:
The only benefit fancy-log gives is it's the same logger used by Gulp, so we get date and time output formatting consistent with the rest of the build output.  It's a tiny nice-to-have, I don't particularly care either way.

Reply:
Don't have stong feelings either, up to you.